modules/update-using-custom-machine-config-pools-mcp-remove.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * updating/update-using-custom-machine-config-pools.adoc
+
+[id="update-using-custom-machine-config-pools-mcp-remove_{context}"]
+= Moving a node to the original machine config pool
+
+In this canary rollout update process, after you have unpaused a custom machine config pool (MCP) and verified that the applications on the nodes associated with that MCP are working as expected, you should move the node back to its original MCP by removing the custom label you added to the node.
+
+[IMPORTANT]
+====
+A node must have a role to be properly functioning in the cluster.
+====
+
+To move a node to its original MCP
+
+. Ensure that the nodes have a `worker` label or a label from an MCP that is updated.
++
```

Comment:
It seems this step not related with this section. We can just jump to step 2 for `Moving a node to the original machine config pool`. And in previous operations, we don't have steps to remove the `worker` label from any node.